packages/longhorn/longhorn.patch

```diff
@@ -0,0 +1,12 @@
+diff -x '*.tgz' -x '*.lock' -uNr packages/longhorn/charts-original/Chart.yaml packages/longhorn/charts/Chart.yaml
+--- packages/longhorn/charts-original/Chart.yaml
++++ packages/longhorn/charts/Chart.yaml
+@@ -25,3 +25,8 @@
+ - https://github.com/longhorn/longhorn-ui
+ - https://github.com/longhorn/longhorn-tests
+ version: 1.0.1
+++annotations:
++  catalog.rancher.io/certified: rancher
++  catalog.rancher.io/experimental: true
```

Comment:
Is it really still experimental @yasker?

Reply:
No, it's no longer experimental.